module/documents/chat-message.mjs

```diff
@@ -555,6 +551,26 @@ export default class ChatMessage5e extends ChatMessage {
     afterElement.insertAdjacentElement("beforebegin", enchantmentApplication);
   }
 
+  /* -------------------------------------------- */
+
+  /**
+   * Display the effects tray with effects the user can apply.
+   * @param {HTMLLiElement} html  The chat card.
+   * @protected
+   */
+  _enrichUsageEffects(html) {
+    const item = this.getAssociatedItem();
+    const effects = this.getFlag("dnd5e", "use.effects")
```

Comment:
We should also make sure that Items that are posted directly to chat (i.e. when they have no activities) also still show applicable effects.

Reply:
Is that something we want to handle? Because with activities you explicitly define what effects to include, but for posting to chat it will just put all of them?

Reply:
This would only be for items that don't have any activities anyway, so their effects can't be associated with anything but the item.